docs/src/pages/basics/live-examples/index.md

```diff
@@ -5,55 +5,47 @@ title: 'Live Examples'
 
 ### Next release
 
-- [React Official](https://storybookjs-next.now.sh/official-storybook/)
-- [Vue](https://storybookjs-next.now.sh/vue-kitchen-sink/)
-- [Angular](https://storybookjs-next.now.sh/angular-cli/)
-- [Mithril](https://storybookjs-next.now.sh/mithril-kitchen-sink/)
-- [Marko](https://storybookjs-next.now.sh/marko-cli/)
-- [HTML](https://storybookjs-next.now.sh/html-kitchen-sink/)
-- [Svelte](https://storybookjs-next.now.sh/svelte-kitchen-sink/)
-- [Riot](https://storybookjs-next.now.sh/riot-kitchen-sink/)
-- [Ember](https://storybookjs-next.now.sh/ember-cli/)
-- [Preact](https://storybookjs-next.now.sh/preact-kitchen-sink/)
-- [Rax](https://storybookjs-next.now.sh/rax-kitchen-sink/)
+- [React Official](https://next--storybookjs.netlify.com/official-storybook/)
+- [Vue](https://next--storybookjs.netlify.com/vue-kitchen-sink/)
+- [Angular](https://next--storybookjs.netlify.com/angular-cli/)
+- [Mithril](https://next--storybookjs.netlify.com/mithril-kitchen-sink/)
+- [Marko](https://next--storybookjs.netlify.com/marko-cli/)
+- [HTML](https://next--storybookjs.netlify.com/html-kitchen-sink/)
+- [Svelte](https://next--storybookjs.netlify.com/svelte-kitchen-sink/)
+- [Riot](https://next--storybookjs.netlify.com/riot-kitchen-sink/)
+- [Ember](https://next--storybookjs.netlify.com/ember-cli/)
+- [Preact](https://next--storybookjs.netlify.com/preact-kitchen-sink/)
+- [Rax](https://next--storybookjs.netlify.com/rax-kitchen-sink/)
 
 ### Current release
 
-- [React Official](https://storybookjs.now.sh/official-storybook/)
-- [Vue](https://storybookjs.now.sh/vue-kitchen-sink/)
-- [Angular](https://storybookjs.now.sh/angular-cli/)
-- [Mithril](https://storybookjs.now.sh/mithril-kitchen-sink/)
-- [Marko](https://storybookjs.now.sh/marko-cli/)
-- [HTML](https://storybookjs.now.sh/html-kitchen-sink/)
-- [Svelte](https://storybookjs.now.sh/svelte-kitchen-sink/)
-- [Riot](https://storybookjs.now.sh/riot-kitchen-sink/)
-- [Ember](https://storybookjs.now.sh/ember-cli/)
-- [Preact](https://storybookjs.now.sh/preact-kitchen-sink/)
-- [Rax](https://storybookjs.now.sh/rax-kitchen-sink/)
-
-### 5.1
-
-- [React Official](https://storybooks-official.netlify.com)
-- [Vue](https://storybooks-vue.netlify.com/)
-- [Angular](https://storybooks-angular.netlify.com/)
-- [Polymer](https://storybooks-polymer.netlify.com/)
-- [Mithril](https://storybooks-mithril.netlify.com/)
-- [Marko](https://storybooks-marko.netlify.com/)
-- [HTML](https://storybooks-html.netlify.com/)
-- [Svelte](https://storybooks-svelte.netlify.com/)
-- [Riot](https://storybooks-riot.netlify.com/)
-- [Ember](https://storybooks-ember.netlify.com/)
-- [Preact](https://storybooks-preact.netlify.com/)
+- [React Official](https://storybookjs.netlify.com/official-storybook/)
```

Comment:
@ndelangen these links don't work. is there another PR we need to patch back to `master`?

Reply:
Going to investigate

Reply:
Going to fix this asap:
https://app.netlify.com/sites/storybookjs/deploys/5e3a8b3054f52300084af24c